src/components/IndexPage/hero.js

```diff
@@ -109,23 +169,56 @@ const Arrow = ({ color, scale }) => {
   )
 }
 
-const Banner = () => <ImgContainer data={banner}>Banner</ImgContainer>
-
-const Hero = ({ title, background }) => {
-  return (
-    <StyledBackgroundImage
-      fluid={background.childImageSharp.fluid}
-      backgroundColor={`#040e18`}
-    >
-      <Section>
-        <Banner dropShadow />
-        <Subtitle>{title}</Subtitle>
-        <br />
+const Hero = ({ title, background }) => (
+  <StaticQuery
+    query={graphql`
+      query Marketing {
+        allMarketingJson {
+          edges {
+            node {
+              event {
+                date
+                time
+                location
+                title
+              }
+              post {
+                title
+                author
+                url
+              }
+            }
+          }
+        }
+      }
+    `}
+    render={data => (
+      <StyledBackgroundImage
+        fluid={background.childImageSharp.fluid}
+        backgroundColor={`#040e18`}
+      >
+        <Section>
+          <IntroductionContainer>
+            <Title>SANDBOX</Title>
+            <Subtitle>{title}</Subtitle>
```

Comment:
I think this should say "Northeastern's student-led software consultancy" our mission will be on the banner below

Reply:
aight changed :))